apps/www/pages/privacy.mdx

```diff
@@ -331,9 +332,9 @@ If you would like to find out more about cookies and other similar technologies,
 
 Please note that deleting or blocking cookies may not be effective for all types of tracking technologies, such as Local Storage Objects (LSOs) like HTML5.
 
-**Cookiesless analytics**
+#### **Cookiesless analytics**
 
-We also use cookieless technologies to collect information to assess how our Service is used. When you access a website or service on the internet, your device and your browser send certain information to the provider of that website or service in order to access that website or service. We collect, aggregate and analyze this information to identify errors in and improvements to Service. We will only do so, however, only if you give us your consent to do so.
+We also use cookieless technologies to collect information to assess how our Service is used. When you access a website or service on the internet, your device and your browser send certain information to the provider of that website or service in order to access that website or service. We collect, aggregate and analyze this information to identify errors in and improvements to Service. We will only do so, however, only if you give us your consent to do so. <PrivacySettings className="underline">Manage preferences</PrivacySettings>
```

Comment:
alternative could be using queryParam for triggering opening or closing settings. 

then its just something like `<a href="?privacySettings=true">`

Reply:
dont worry about it though